package/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "omicron-package"
+version = "0.1.0"
+edition = "2018"
+license = "MPL-2.0"
+
+[dependencies]
+anyhow = "1.0"
+omicron-common = { path = "../common" }
+propolis-server = { git = "https://github.com/oxidecomputer/propolis", rev = "00ec8cf18f6a2311b0907f0b16b0ff8a327944d1" }
```

Comment:
Perhaps include a comment for passersby to note why the `propolis-server` dep is present, since it's not intuitive on first glance?

Reply:
Done